src/LibraryViewExtensionMSWebBrowser/Handlers/NodeItemDataProvider.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dynamo.Search;
+using Dynamo.Search.SearchElements;
+using Dynamo.LibraryViewExtensionMSWebBrowser.Handlers;
+using Newtonsoft.Json;
+
+// TODO there are many clases in this file which are duplicates in LibraryViewExtension - 
+// these can be refactored out into a shared core.
```

Comment:
Thanks, indeed. Will need a tech debt for this.

Reply:
filed